src/iterative_sorting/iterative_sorting.py

```diff
@@ -1,30 +1,44 @@
-# TO-DO: Complete the selection_sort() function below 
-def selection_sort( arr ):
+# TO-DO: Complete the selection_sort() function below
+def selection_sort(arr):
     # loop through n-1 elements
     for i in range(0, len(arr) - 1):
         cur_index = i
         smallest_index = cur_index
         # TO-DO: find next smallest element
-        # (hint, can do in 3 loc) 
-             
-
-
+        # (hint, can do in 3 loc)
+        while cur_index < len(arr):
```

Comment:
Awesome work! Another for loop will work here as well but I like the readability here.